sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -87,6 +98,7 @@ public PHPSensor(ResourcePerspectives resourcePerspectives, FileSystem fileSyste
     this.resourcePerspectives = resourcePerspectives;
     this.fileLinesContextFactory = fileLinesContextFactory;
     this.fileSystem = fileSystem;
+    this.settings = settings;
```

Comment:
what is `sensorContext()`? why method call? You should use parameter passed to `execute` method